lib/runESLint.js

```diff
@@ -29,17 +30,21 @@ const runESLint = ({ fix = false, paths }) =>
     } else {
       console.log(gray(`Paths: ${filteredFilePaths.join(' ')}`));
       try {
-        const { errorCount, warningCount, results } = cli.executeOnFiles(
-          filteredFilePaths,
-        );
+        const report = cli.executeOnFiles(filteredFilePaths);
 
-        if (errorCount || warningCount) {
-          const formatter = cli.getFormatter();
-          console.log(formatter(results));
-        }
+        if (fix) {
```

Comment:
This change means when running in `fix` mode we don't exit with errors if something is not fixable.